### PR TITLE
feat(1.2.1): added GET api endpoint for posts

### PR DESCRIPTION
**Pull request by jxkhan**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ try {
     });
   } catch (error) {
     console.error('Error starting server:', error);
-    process.exit(1);
+    process.exit(2);
```
Comment (jxkhan) [inline app.js]: I will remove this.

File: controllers/postsControllers.js [hunk 2]
```diff
@@ -33,4 +35,15 @@ const newPost= asyncHandler(async(req,res) => {
 
 })
 
-module.exports = newPost
\ No newline at end of file
+  const getPosts= asyncHandler(async(req,res) => {
+  try {
```
Comment (jxkhan) [inline controllers/postsControllers.js:21]: I will use error handler here.

**Review by jxkhan — COMMENTED** [review]
A couple of observations. 
I will also add timestamps.

File: controllers/postsControllers.js [hunk 3]
```diff
@@ -1,49 +1,46 @@
...
+    return res.status(500).json({ error: error });
+  }
+});
+
+const getPosts = asyncHandler(async (req, res) => {
+  console.log("Fetching posts...");
+  const posts = await Posts.findAll();
+  console.log("Posts fetched:", posts);
+  res.json(posts);
+  if(!posts)
```
Comment (jxkhan) [inline controllers/postsControllers.js:40]: error handler added

File: app.js [hunk 4]
```diff
@@ -12,7 +12,7 @@ try {
     });
   } catch (error) {
     console.error('Error starting server:', error);
-    process.exit(2);
+    process.exit(1);
```
Comment (jxkhan) [inline app.js]: fixed

File: models/postsModel.js [hunk 5]
```diff
@@ -34,6 +34,16 @@ const Posts = sequelize.define(
       type: DataTypes.STRING,
       references: "image_path"
     },
+    created_at: {
```
Comment (jxkhan) [inline models/postsModel.js:37]: time stamps added

**Review by xalkan — APPROVED** [review]

**Pull request merged**